packages/vue/src/components/organisms/SfStoreLocator/_internal/SfStore.vue

```diff
@@ -6,15 +6,17 @@
         <span>{{ distance }}km</span> away from you
       </div>
     </slot>
-    <div class="sf-store__media" @click="centerOn(latlng)">
+    <div v-bind:class="{ 'sf-store__media': picture }" @click="centerOn(latlng)">
       <!-- @slot Use this slot to show media elements -->
       <slot name="media">
         <SfImage
+          v-if="picture"
           :src="picture"
           :alt="`${name} picture`"
           :width="82"
           :height="112"
         />
+        <SfIcon v-if="!picture" icon="marker" />
```

Comment:
```suggestion
        <SfIcon v-else icon="marker" />
```

Is `v-else` better in that place. isn't it?

Reply:
@Michal-Dziedzinski Thanks for the valid suggestion. I'll update the PR according to it.